### DISCUSSION TITLE
Time Elapsed Since Last Successful Seed

### DISCUSSION THREAD

**Discussion by calamityoblivion** [Zodsmar/SeedSearcherStandaloneTool#103] [Ideas]
Could you please add the time elapsed since the last successful found seed, because it could help people more accurately know approximately how long it takes to find each seed so that they can widen or narrow what they are looking for when trying to find multiple seeds under certain specifications. Thank you.

**Comment by Zodsmar** [comment]
Ill look to add this in the next version

**Comment by Zodsmar** [comment]
I added this for the next version 
5a48cf3
